+vis/emptyLayer.m

```diff
@@ -17,17 +17,17 @@
 % practice.
 layer.textureId = [];
 % POS TODO
-layer.pos = [0 0];
+layer.pos = [0 0]';
```

Comment:
All these transposes are unnecessary.  I'd try to avoid defining a vector then immediately transposing it.  Obviously not a big deal...